thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -243,6 +243,8 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
             builder.clear();
             if (builderIFileIO)
             {
+                stat_type numDiskWrites = builderIFileIO->getStatistic(StNumDiskWrites);
+                stats.sumStatistic(StNumDiskWrites, numDiskWrites);
```

Comment:
I think you want a more general:

                mergeStats(stats, builderIFileIO, diskWriteRemoteStatistics);

And a change to indexWriteActivityStatistics below.